aspnetcore/fundamentals/middleware/sample/Chain/Startup.cs

```diff
@@ -0,0 +1,29 @@
+﻿#define PrimarySample
```

Comment:
@JunTaoLuo I had several customers complain it was very confusing. They prefer the standalone and Startup. They can just copy/paste the sample code in the article and it works without having to figure out config.